src/customers/customer-groups.md

```diff
@@ -4,7 +4,7 @@ title: Customer Groups
 
 Customer groups determine which discounts are available and the tax class that is associated with the group. The default customer groups are General, Not Logged In, and Wholesale.
 
-- {:.b2b-only}The selection of customer groups includes all regular customer groups, and shared catalogs, even if [Shared Catalogs]({% link stores/b2b-features.md %}) is not enabled in the configuration. Only one customer group or shared catalog can be assigned to a [company]({% link customers/account-companies.md %}) at a time.
+- {:.b2b-only}The selection of customer groups includes all regular customer groups, and shared catalogs, even if [Shared Catalogs]({% link catalog/catalog-shared.md %}) is not enabled in the configuration. Only one customer group or shared catalog can be assigned to a [company]({% link customers/account-companies.md %}) at a time.
```

Comment:
I see why you want this changed. It is confusing. 

After some research, I think the intention is to draw your attention to enabling the Shared Catalog feature, which is in step 3 of the Configure B2B features topic. What do you think?

Perhaps:

```suggestion
- {:.b2b-only}The selection of customer groups includes all regular customer groups and shared catalogs, even if **Shared Catalogs** is not enabled in the configuration. See [Configure B2B features]({% link stores/b2b-features.md %}) to enable shared catalogs. Only one customer group or shared catalog can be assigned to a [company]({% link customers/account-companies.md %}) at a time.
```

Reply:
Hi @hguthrie fun fact, on the Shared Catalogs page there is also a link that leads to the page of all B2B functions, so the customer should get there anyway (I hope😁)
![image](https://user-images.githubusercontent.com/78729171/116451216-e3faea00-a864-11eb-9715-502ac2059d4e.png)
But I completely trust your experience, and I like your version. I will implement this with the next commit
Many thanks ☺️

Reply:
Done @hguthrie, I think looks great, thank you!

Could you please check and approve?